tests/acceptance/features/webUIPreview/imageMediaViewer.feature

```diff
@@ -78,3 +78,15 @@ Feature: display image in media viewer on the webUI
     And user "user1" has logged in using the webUI
     When the user views the file "testimage.mp3" in the media viewer using the webUI
     Then the file "testimage.mp3" should be displayed in the media viewer webUI
+
+  Scenario: preview of png file with media viewer is possible
+    Given user "user1" has uploaded file "testavatar.png" to "testavatar.png"
+    And user "user1" has logged in using the webUI
+    When the user views the file "testavatar.png" in the media viewer using the webUI
+    Then the file "testavatar.png" should be displayed in the media viewer webUI
```

Comment:
```suggestion
  Scenario Outline: preview of an image file with media viewer is possible
    Given user "user1" has uploaded file "<image>" to "<image>"
    And user "user1" has logged in using the webUI
    When the user views the file "<image>" in the media viewer using the webUI
    Then the file "<image>" should be displayed in the media viewer webUI
    Examples:
    | image           |
    | testavatar.png  |
    | testavatar.jpeg |
```
Make this scenario a `Scenario Outline`. Then you do not need 2 scenarios.

Reply:
there's another scenario for .jpg file as well. Since they're all image files, is it okay if I merge them all into one ?

Reply:
if it tests a similar action then you can merge them together.